src/rust/cryptography-x509-validation/src/lib.rs

```diff
@@ -10,3 +10,14 @@ pub mod ops;
 pub mod policy;
 pub mod trust_store;
 pub mod types;
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum ValidationError {
+    Other(String),
+}
+
+impl From<&str> for ValidationError {
```

Comment:
Not a fan of `From<&str>` it's way too non-specific and makes the code hard to read

Reply:
Ok, removing.